docs/contents/hooks/use-theme.ja.mdx

```diff
@@ -33,19 +33,36 @@ return (
     <Text>The current scheme is "{themeScheme}"</Text>
 
     <Wrap mt="md" gap="md">
-      <Button onClick={() => changeThemeScheme("base")}>Base Theme</Button>
-
-      <Button colorScheme="pink" onClick={() => changeThemeScheme("pink")}>
-        Pink Theme
-      </Button>
-
-      <Button colorScheme="purple" onClick={() => changeThemeScheme("purple")}>
-        Purple Theme
-      </Button>
-
-      <Button colorScheme="green" onClick={() => changeThemeScheme("green")}>
-        Green Theme
-      </Button>
+      <For
+        each={[
+          { changeThemeScheme: "base", children: "Base Theme" },
+          {
+            colorScheme: "pink",
+            changeThemeScheme: "pink",
+            children: <>Pink Theme</>,
+          },
+          {
+            colorScheme: "purple",
+            changeThemeScheme: "purple",
+            children: <>Purple Theme</>,
+          },
+          {
+            colorScheme: "green",
+            changeThemeScheme: "green",
+            children: <>Green Theme</>,
+          },
+        ]}
+      >
+        {(item, index) => (
+          <Button
+            key={index}
+            colorScheme={item.colorScheme}
+            onClick={() => changeThemeScheme(item.changeThemeScheme)}
+          >
+            {item.children}
+          </Button>
+        )}
```

Comment:
I think it would be better to align the documentation and style with [the other components](https://github.com/yamada-ui/yamada-ui/pull/3978/files).



```suggestion
        {({colorScheme, changeThemeScheme, children}, index) => (
          <Button
            key={index}
            colorScheme={colorScheme}
            onClick={() => changeThemeScheme(changeThemeScheme)}
          >
            {children}
          </Button>
        )}
```